osmdroid-android/src/main/java/org/osmdroid/views/Projection.java

```diff
@@ -703,36 +722,49 @@ private void adjustOffsets(final long pDeltaX, final long pDeltaY) {
 	}
 
 	/**
+	 * @param pPixelMin Pixel position of the limit (left)
```

Comment:
so this is public, why would someone want to call this? not clear (to me) based on the method name or the javadocs

Reply:
It's public only in order to be accessed by `ProjectionTest`.

Reply:
would protected work for this?

Reply:
Doesn't work for `ProjectionTest` if `protected` in `Projection`.